docs/installation/SUSE.md

```diff
@@ -95,7 +101,10 @@ and volumes run the following command:
 
 You must delete the user created configuration files manually.
 
-## What's next
+# What's next
 
 Continue with the [User Guide](/userguide/).
 
+More details about Docker on openSUSE or SUSE Linux Enteprise can be
+found [here](https://www.suse.com/documentation/sles-12/dockerquick/data/dockerquick.html).
```

Comment:
I think we only need the first sentence, because we already mention "openSUSE" and "SUSE Linux Enterprise". I also think we should change the link "caption" to something more descriptive, perhaps;

```Markdown
You can find more details about Docker on openSUSE or SUSE Linux Enterprise in the
[Docker quick start guide](https://www.suse.com/documentation/sles-12/dockerquick/data/dockerquick.html)
on the SUSE website.
```